### PR TITLE
[izumi-iziswap]update poolHelpers contracts

### PR DESCRIPTION
**Pull request by LpcAries**
We have updated the poolHelpers contracts on some chains, so we have adjusted the code to simultaneously detect TVL on both new and old contracts.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/izumi-iziswap exports TVL: 
        
 
 
    ethereum                  0
    meter                     80.02 k
    arbitrum                  10.81 k
    cronos                    2.0479588800984874
    ultron                    223.72 k
    mantle                    378.73 k
    polygon                   348.46794168457
    aurora                    387.79 k
    era                       16.62 M
    ontology_evm              18.09 k
    bsc                       1.12 M
    
    total                    18.84 M

File: projects/izumi-iziswap/index.js [hunk 1]
```diff
@@ -27,40 +30,48 @@ const blacklistedTokens = [
...
-      target: poolHelpers[chain],
-      abi: abi.poolMetas,
-      calls,
-    })
-    for (const output of poolMetas) {
-      if (output.tokenX === nullAddress && output.fee === '0') {
-        foundLastPool = true
-        break;
+  for(const manager of poolHelpers[chain]) {
+    console.info(manager)
```
Comment (g1nt0ki) [inline projects/izumi-iziswap/index.js:40]: code looks good, remove remove the console line

**Comment by llamatester** [comment]
The adapter at projects/izumi-iziswap exports TVL: 
        
 
 
    ethereum                  0
    arbitrum                  10.81 k
    meter                     80.29 k
    ultron                    223.57 k
    cronos                    2.0484121257531007
    polygon                   348.5020337221649
    mantle                    378.75 k
    aurora                    386.70 k
    era                       16.53 M
    bsc                       1.12 M
    ontology_evm              17.95 k
    
    total                    18.75 M

File: projects/izumi-iziswap/index.js (hunk 1 above)
Comment (LpcAries) [reply projects/izumi-iziswap/index.js]: Sorry, my miss. Code updated.

**Pull request merged**